DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -191,6 +196,12 @@ export class SearchSortFilterAndPaginate {
     return element.getAttribute('data-filter-value')?.trim() ?? '';
   }
 
+  private displaySearchableElementsAndRunSuppliedFunction(searchableElements: ISearchableElement[])

```

Comment:
I think 'runPostDisplayFunction' or something like that would be clearer than 'runSuppliedFunction'.

Reply:
Good call. I've renamed it.